instrumentation-api/build.gradle.kts

```diff
@@ -1,6 +1,5 @@
 plugins {
   id("org.xbib.gradle.plugin.jflex")
-
```

Comment:
Can you revert this change? We don't have a strict policy but generally group built-in Gradle plugins, third party plugins, and our own plugins